zoobot/pytorch/training/train_with_pytorch_lightning.py

```diff
@@ -184,36 +178,28 @@ def loss_func(preds, labels):  # pytorch convention is preds, labels
     logging.info((trainer.training_type_plugin, trainer.world_size,
                  trainer.local_rank, trainer.global_rank, trainer.node_rank))
 
+    # TEMP
     trainer.fit(lightning_model, datamodule)
 
-    trainer.test(
-        model=lightning_model,
-        datamodule=datamodule,
-        ckpt_path='best'  # can optionally point to a specific checkpoint here e.g. "/share/nas2/walml/repos/gz-decals-classifiers/results/early_stopping_1xgpu_greyscale/checkpoints/epoch=26-step=16847.ckpt"
-    )
+    # trainer.test(
```

Comment:
any reason to remove the test step of after fitting? Does this speed up the system but impact quality?

Reply:
Testing routinely is bad practice as you may accidentally tune your hparams to overfit the model. I was being a bit lazy when adding here earlier. I have left it commented as example, with a warning note.